segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -1069,6 +1074,10 @@ public void readTableEntriesDelta(WireCommands.ReadTableEntriesDelta readTableEn
     private void truncateIndexSegment(String segment, long offset) {
         String indexSegment = getIndexSegmentName(segment);
         try {
+            if (!isUserStreamSegment(segment)) {
+                log.debug("No need to perform truncation of index segment for {}.", segment);
```

Comment:
This logs can be of trace level, when debug logs enabled it will be printed for all inter segments which will populate the logs.

Reply:
Truncation of segment is not a frequent operation, so i think we can keep it at info level.